### PR TITLE
Added ro-RO locale file

### PR DESCRIPTION
**Pull request by cristake**
Hello,
I've added support for romanian language, hope this helps.

### CODE REVIEW

**Comment by djhvscf** [comment]
Can you add the "normal" version.. not only the min version.. please :)

**Comment by cristake** [comment]
I have just added the normal version of ro-RO :)

File: src/locale/bootstrap-table-ro-RO.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Bootstrap Table English translation
```
Comment (djhvscf) [inline src/locale/bootstrap-table-ro-RO.js]: Fix the title please

File: dist/locale/bootstrap-table-ro-RO.min.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*
+* bootstrap-table - v1.6.0 - 2015-03-03
```
Comment (djhvscf) [inline dist/locale/bootstrap-table-ro-RO.min.js]: Fix this information please

**Comment by cristake** [comment]
fixed info

**Pull request merged**